dev-python/flask-paginate/flask-paginate-0.7.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${PV}.tar.gz -> ${P}-${PV}.tar.gz"
```

Comment:
${P} variable in this case is the same as ${PN}-${PV}. Please leave it as ${P}.tar.gz

Reply:
I know and understand this, but could you please explain to me, why i'm then getting: "The source directory ${S} doesn't exist" ?

Reply:
Should we define a $S ?

Reply:
I'm not getting those errors. Check which source directory is defined and the directory name on PORTAGE_TMPDIR

Reply:
Tarball name has nothing to do with `S` structure. You can inspect the tarball by simply opening/extracting it, and/or browse to `PORTAGE_TMPDIR` as suggested when emerge fails.